numpy/core/include/numpy/ufuncobject.h

```diff
@@ -209,6 +209,9 @@ typedef struct _tagPyUFuncObject {
          * set by nditer object.
          */
         npy_uint32 iter_flags;
+
+        /* Identity for reduction, when identity == PyUFunc_IdentityValue */
+        PyObject *identity_value;
```

Comment:
This breaks ABI compatibility, doesn't it...

Reply:
?

Reply:
Adding a new field changes.`sizeof(PyUfuncObject)`, which as I understand it could be a breaking abi change.